src/neighbor.cpp

```diff
@@ -2582,60 +2580,58 @@ void Neighbor::modify_params(int narg, char **arg)
       iarg += 2;
     } else if (strcmp(arg[iarg],"once") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal neigh_modify command");
+      if (iarg+2 > narg) error->all(FLERR, "Invalid once keyword: expected 1 once argument");
       build_once = utils::logical(FLERR,arg[iarg+1],false,lmp);
       iarg += 2;
     } else if (strcmp(arg[iarg],"page") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal neigh_modify command");
+      if (iarg+2 > narg) error->all(FLERR, "Invalid page keyword: expected 1 page argument");
       old_pgsize = pgsize;
       pgsize = utils::inumeric(FLERR,arg[iarg+1],false,lmp);
       iarg += 2;
     } else if (strcmp(arg[iarg],"one") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal neigh_modify command");
+      if (iarg+2 > narg) error->all(FLERR, "Invalid one keyword: expected 1 one argument");
       old_oneatom = oneatom;
       oneatom = utils::inumeric(FLERR,arg[iarg+1],false,lmp);
       iarg += 2;
     } else if (strcmp(arg[iarg],"binsize") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal neigh_modify command");
+      if (iarg+2 > narg) error->all(FLERR, "Invalid binsize keyword: expected 1 binsize argument");
       binsize_user = utils::numeric(FLERR,arg[iarg+1],false,lmp);
       if (binsize_user <= 0.0) binsizeflag = 0;
       else binsizeflag = 1;
       iarg += 2;
     } else if (strcmp(arg[iarg],"cluster") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal neigh_modify command");
+      if (iarg+2 > narg) error->all(FLERR, "Invalid cluster keyword: expected 1 cluster argument");
       cluster_check = utils::logical(FLERR,arg[iarg+1],false,lmp);
       iarg += 2;
-
     } else if (strcmp(arg[iarg],"include") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal neigh_modify command");
+      if (iarg+2 > narg) error->all(FLERR, "Invalid include keyword: expected 1 include argument");
       includegroup = group->find(arg[iarg+1]);
-      if (includegroup < 0)
-        error->all(FLERR,"Invalid group ID in neigh_modify command");
-      if (includegroup && (atom->firstgroupname == nullptr ||
-                            strcmp(arg[iarg+1],atom->firstgroupname) != 0))
-        error->all(FLERR,
-                   "Neigh_modify include group != atom_modify first group");
+      if (includegroup == -1)
+        error->all(FLERR, "Invalid include keyword: group {} not found", arg[iarg+1]);
+      if (atom->firstgroupname == nullptr)
+          error->all(FLERR, "Invalid include keyword: atom_modify first command must be used");
+      if (strcmp(arg[iarg+1],atom->firstgroupname) != 0)
+        error->all(FLERR, "Neigh_modify include group != atom_modify first group: {}", atom->firstgroupname);
       iarg += 2;
-
     } else if (strcmp(arg[iarg],"exclude") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal neigh_modify command");
+      if (iarg+2 > narg) error->all(FLERR, "Invalid exclude keyword: loss exclude arguments");
```

Comment:
I don't know what this message means, but see my previous comment about using `utils::missing_cmd_args()` for all these cases where arguments are missing.